CHANGELOG.md

```diff
@@ -59,6 +59,25 @@ This change was made in [pull request #3819: Add linked image focus style](https
 
 You must make the following changes when you migrate to this release, or your service might break.
 
+#### Update component initialisation
+
+We have removed the `init()` method from all components which include JavaScript. Initialisation now happens when the
+component is created.
+
+If you import the JavaScript using `window.GOVUKFrontend.initAll()`, you will not need to make any changes.
+
+If you initialise any component individually, you will need to remove the `.init()`. For example:
+
+```js
```

Comment:
Super subtle, but can we use `mjs` by default just in case we turn ESLint back on for CHANGELOG?

```suggestion
```mjs
```